tests/regression/runner.py

```diff
@@ -28,13 +27,19 @@
 MSG_STATUS_CODE_400 = "{\"message\":\"Invalid request header\"}"
 MSG_STATUS_CODE_404 = "404 Not Found"
 MSG_STATUS_CODE_500 = "500"
+MSG_INVALID_PATH = "{\"message\":\"Invalid path\"}"
+STATUS_CODE_500 = "500"
```

Comment:
Why are there duplicate status codes?

Reply:
@HowJMay This is still unresolved.

Reply:
It seems github's problem. The code in "Files changed" is the modified one.

Reply:
I mean not just `500`. Why other status have so many duplicate variables. There shall be a better way than this.